modules/integration/src/test/scala/scala/cli/integration/FixTests.scala

```diff
@@ -0,0 +1,124 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+class FixTests extends ScalaCliSuite {
```

Comment:
What happens if I now have directives in different scopes? For example main directives in `Hello.scala` and test directives in `Hello.test.scala`?

We should make sure that we don't accidentally broke someone's Scala CLI configuration.

Reply:
It should be also tested with Scala.Js and Scala native scopes.